testsuite/integration-arquillian/test-apps/cors/angular-product/src/main/webapp/keycloak.json

```diff
@@ -1,8 +1,9 @@
 {
   "realm" : "cors",
   "realm-public-key" : "MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQCrVrCuTtArbgaZzL1hvh0xtL5mc7o0NqPVnYXkLvgcwiC3BjLGw1tGEGoJaXDuSaRllobm53JBhjx33UNv+5z/UMG4kytBWxheNVKnL6GgqlNabMaFfPLPCF8kAgKnsi79NMo+n6KnSY8YeUmec/p2vjO2NjsSAVcWEQMVhJ31LwIDAQAB",
-  "auth-server-url" : "http://localhost-auth:8180/auth",
+  "auth-server-url" : "https://localhost-auth:8543/auth",
   "ssl-required" : "external",
+  "disable-trust-manager": true,
```

Comment:
We shouldn't disable trust manager. One of the main points on SSL testing is that we make sure certs are verified properly and we don't end up allowing untrusted certificates automatically. Allowing untrusted certificates should only be possible if explicitly enabled.